src/cli/cli.cc

```diff
@@ -3627,6 +3700,86 @@ int main (const int argc, const char* argv[]) {
       exit(exitCode);
     }
   });
+  
+  createSubcommand("setup", { "--platform", "--yes" }, false, [&](const std::span<const char *>& options) -> void {
```

Comment:
`"-y"` is missing in a vector of valid options. CLI will print an error that `"-y"` is not valid.

Reply:
Great spot.